lib/view_component/storybook/stories.rb

```diff
@@ -12,6 +12,10 @@ class Stories
       validate :valid_story_configs
 
       class << self
+        def stories_title(title = nil)
+          self.title = title unless title.nil?
+        end
```

Comment:
is there a reason to allow this to be called without any params?

Reply:
So actually no – I followed the `def layout` format incorrectly, will update